x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.test.ts

```diff
@@ -801,6 +801,58 @@ describe('data telemetry collection tasks', () => {
     });
   });
 
+  describe('custom dashboards', () => {
+    const task = tasks.find((t) => t.name === 'custom_dashboards');
+    const savedObjectsClient = savedObjectsClientMock.create();
+
+    it('returns custom dashboards stats from all spaces', async () => {
+      savedObjectsClient.find.mockResolvedValueOnce({
+        page: 1,
+        per_page: 500,
```

Comment:
500/page could potentially explode the response size. Please watch that and reduce it if the response size starts to cause issues.